raft_test.go

```diff
@@ -58,6 +58,7 @@ func TestRaftStability_Large_Values(t *testing.T) {
 	var wg sync.WaitGroup
 	var numStarted uint32
 	for _, data := range dataBlocks {
+		data := data
 		// Should be able to apply
 		wg.Add(1)
 		go func() {
```

Comment:
feel free to ignore as it's in a testing file but a more conventional way to fix this warning would be to pass `data` as a param to the anonymous func.

Reply:
I can do that. This was just me blindly following the linter recommendation but passing it as an arg to the function seems better.